pkg/features/features_test.go

```diff
@@ -57,24 +57,38 @@ func testFlagEnabled(t *testing.T, feature FeatureFlag, test envTest, defaultVal
 		}
 
 		got := feature.Enabled()
-		if buildinfo.ReleaseBuild {
+		if buildinfo.ReleaseBuild && unchangeableFeature {
 			assert.Equal(t, got, defaultValue)
 		} else {
 			assert.Equal(t, got, test.expected)
 		}
 	})
 }
 
-func TestFlags(t *testing.T) {
+func TestFeatureFlags(t *testing.T) {
```

Comment:
What I'm missing is the mechanism through which the feature is changed in the runtime. Is it done through `os.Setenv()`? Why is that not part of the `FeatureFlag` contract? How is the value persisted between restarts?

Reply:
Figured this one out.